packages/storybook/src/nimble/wafer-map/wafer-map-matrix.stories.ts

```diff
@@ -1,8 +1,8 @@
 import type { StoryFn, Meta } from '@storybook/html';
 import { html, ViewTemplate } from '@microsoft/fast-element';
+import { tableFromArrays } from 'apache-arrow';

```

Comment:
It's great that we're getting really close to parity between the experimental wafer map and the original one. I visually compared the stories in [the storybook build](https://60e89457a987cf003efc0a5b-vfitydsqhq.chromatic.com/?path=/story/tests-wafer-map--experimental-wafer-map-grid-dimensions-test) and noticed a few differences:
1. In "Wafer map dies and orientation test" there are slightly different colors on the die labeled "15" (might need to use an image sampling tool to see the difference)
2. "Wafer map grid dimensions test" there are some visual artifacts in the experimental version (random rectangles rendered outside the bounds of the wafer)
3. "Wafer map highlighted test" exists only for the original renderer, not the experimental one. Is highlighting still a feature gap?

## Issue 1
**Original**
![image](https://github.com/ni/nimble/assets/10500124/df713c78-bb3d-47d5-ae5b-578fee458b5e)

**Experimental**
![image](https://github.com/ni/nimble/assets/10500124/a61e74ec-2a3e-43c6-a311-00b1974c65b3)

## Issue 2

**Original**
![image](https://github.com/ni/nimble/assets/10500124/f213713d-269d-43e6-8b1a-be082d8170ce)

**Experimental**
![image](https://github.com/ni/nimble/assets/10500124/29126496-4c7c-4c4a-bacf-e1156a5af2ea)

Reply:
Issue 1 is expected to happen because we are taking the approach to slice the color interpolation for the linear color scale to increase performance.
Issue 2 was caused by the fac that the new vertical and horizontal scales did not have domain borders for which to filter out dies outside the bounding box. changed the approach to filter out dies before drawing

Reply:
> "Wafer map highlighted test" exists only for the original renderer, not the experimental one. Is highlighting still a feature gap?

@munteannatan what's the direction we are thinking for highlighting? If that is work we still expect can you track it in https://github.com/ni/nimble/issues/2034 ?

Reply:
we hope to do the highlighting using arquero, but yes, we have to track it